com.playeveryware.eos/Runtime/Core/EOSCommandLineOverrides.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using PlayEveryWare.EpicOnlineServices;
+
+namespace PlayEveryWare.EpicOnlineServices
+{
+    /// <summary>
+    /// Holds override values for Deployment and Sandbox IDs.
+    /// </summary>
+    public static class EOSOverrideState
+    {
+        public static string DeploymentIdOverride { get; set; }
+        public static string SandboxIdOverride { get; set; }
+    }
+}
+/// <summary>
+/// Applies CLI overrides for Epic Online Services before any scene loads.
+/// </summary>
+public static class EOSCommandLineOverrides
+{
+    private const string DeploymentKey = "-epicdeploymentid";
+    private const string SandboxKey = "-epicsandboxid";
+
+    [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.SubsystemRegistration)]
+    private static void ApplyCliOverrides()
+    {
+        try
+        {
+            Debug.Log($"[{nameof(EOSCommandLineOverrides)}::{nameof(ApplyCliOverrides)}] Initializing CLI overrides�");
```

Comment:
Invalid char in message
```suggestion
            Debug.Log($"[{nameof(EOSCommandLineOverrides)}::{nameof(ApplyCliOverrides)}] Initializing CLI overrides");
```

Reply:
Comment applied